.github/workflows/package.yml

```diff
@@ -24,9 +24,26 @@ jobs:
           name: ${{ matrix.os }}
           path: 'custom-prompt/bin/*'
 
+  build_other:
+    name: build on ${{ matrix.arch }}/debian:12
+    runs-on: ubuntu-22.04
+    container: ${{ matrix.arch }}/debian:12
+    strategy:
+      matrix:
+        arch: [i386, amd64]
+    steps:
+      - run: apt update && apt install -y build-essential git libnotify-dev libx11-dev
+      - uses: actions/checkout@v1
+      - run: make -j release
+        working-directory: custom-prompt
+      - uses: actions/upload-artifact@v1
```

Comment:
Mention the reason for using old versions of the checkout and upload actions. Also add these links in a commit message.

* actions/runner #2115
* actions/upload-artifact #361
* actions/checkout #1681

Added a space to prevent them from turning into links.